CHANGELOG.md

```diff
@@ -5,8 +5,30 @@
 [1]: https://pypi.org/project/google-cloud-pubsub/#history
 
 
+## 2.0.0
+
+09-11-2020 05:03 PDT
+
+
+### Implementation Changes
+
+- Transition the library to microgenerator. ([#158](https://github.com/googleapis/python-pubsub/pull/158))
+  This is a **breaking change** that introduces several **method signature changes** and **drops support
+  for Python 2.7 and 3.5**.
```

Comment:
Does the ** after 3.5 mean something special? Perhaps you mean <=3.5?

Reply:
It's just markdown, but somehow a line break sneaked in, will fix.

Reply:
Ah, but it does [render](https://github.com/googleapis/python-pubsub/pull/189/files?short_path=4ac32a7#diff-4ac32a78649ca5bdd8e0ba38b7006a1e) correctly, it's fine then.